bigbluebutton-html5/private/locales/en.json

```diff
@@ -345,6 +344,15 @@
     "app.video.stats.rtt": "RTT",
     "app.video.stats.encodeUsagePercent": "Encode usage",
     "app.video.stats.currentDelay": "Current delay",
+    "app.sfu.mediaServerConnectionError2000": "Error 2000: Unable to connect to media server",
+    "app.sfu.mediaServerOffline2001": "Error 2001: Media server is offline. Please try again later.",
+    "app.sfu.mediaServerNoResources2002": "Error 2002: Media server has no available resources",
+    "app.sfu.mediaServerRequestTimeout2003": , "Error 2003: Media server requests are timing out",
```

Comment:
` ,` in the middle of the line .... `out2003": , "Err`

Reply:
fixed